src/airbash

```diff
@@ -8,41 +7,79 @@
 # access point, thus forcing a reconnect. In order to capture the handshake,
 # airodump is being run after sending the deauthentication packet. Verification
 # of a captured handshake is done using aircrack-ng
+
+# get platform
```

Comment:
Platform checking only needs to be done once, thus the proposed function was changed to an if statement.